src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -85,5 +116,28 @@ private static string ConstructFullRuleIdentifier(ReportingDescriptor reportingD
             }
             return fullRuleIdentifier;
         }
+
+        public static string CreateWorkItemDescription(this SarifLog log)
+        {
+            Dictionary<string, int> resultCountsByTool = FetchToolResultCounts(log);
+            StringBuilder templateText = new StringBuilder(@"This bug has been filed by the Sarif Work Item Automatic Filer.  It contains results for the following tools and issues:");
+            templateText.AppendLine();
+            templateText.AppendLine();
+
+            foreach (KeyValuePair<string, int> toolResult in resultCountsByTool)
+            {
+                templateText.Append(string.Format("*Tool: {0}", toolResult.Key));
+                templateText.AppendLine();
+                templateText.Append(string.Format("     Result count: {0}", toolResult.Value));
+                templateText.AppendLine();
+                templateText.AppendLine();
+            }
+
+            templateText.AppendLine();
+            templateText.Append(@"To see result details, please visit the Scans tab of this bug, or the attached Sarif log.");
```

Comment:
>, please visit the Scans tab of this bug, [](start = 55, length = 41)

Here's an example of why we need some sort of configurable footer. The 'scans' tab is specific to the MS ADO instance, doesn't apply to GH or other ADO instances.

A very simple idea for the MVP, produce your summary using the text below, then inject an optional static string which has been provided to us in the context file.


"This work item was filed for the following issues:\r\n{your summary here}\r\n{inject a configurable footer here if available}"